lib/success.js

```diff
@@ -59,6 +61,12 @@ export default async function success(pluginConfig, context, { Octokit }) {
       logger.log("No commits found in release");
     }
     logger.log("Skip commenting on issues and pull requests.");
+    // TODO: use logger.warn() instead of logger.log()
```

Comment:
There is no `logger.warn` right now, it would need to be added in `semantic-release` core

Reply:
Just a quick fyi to keep in mind: to use the newly added logger in core, we'll need to bump the minimum bound of the peer dependency, which is a breaking change

Reply:
Will do that in a follow-up PR 😉